<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RequeteWindow</class>
 <widget class="QWidget" name="RequeteWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Requête</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Effectuer des requêtes :</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="contactsButton">
           <property name="text">
            <string>Nombre de contacts</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="eventButton">
           <property name="text">
            <string>Evénements entre 2 dates</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="listContactButton">
           <property name="text">
            <string>Liste des choses à faire ou des dates d'événement pour un contact dans un intervalle</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="listContactsButton">
           <property name="text">
            <string>Liste des choses à faire ou des dates d'événement pour tous les contact dans un intervalle</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="frameShape">
          <enum>QFrame::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="page"/>
         <widget class="QWidget" name="page_2">
          <widget class="QLabel" name="nombreContacts">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>70</y>
             <width>251</width>
             <height>81</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_3">
          <widget class="QWidget" name="verticalLayoutWidget_3">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>40</y>
             <width>289</width>
             <height>222</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Evénement entre </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="dateDebutDateEdit"/>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string> et le </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="dateFinDateEdit"/>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <widget class="QPlainTextEdit" name="listeEvenementPlainTextEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="validIntervalButton">
              <property name="text">
               <string>Valider l'intervalle</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="page_4"/>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="quitButton">
     <property name="text">
      <string>Quitter</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
